net-im/skypeforlinux/skypeforlinux-5.1.0.1.ebuild

```diff
@@ -18,36 +20,36 @@ S="${WORKDIR}"
 QA_PREBUILT=opt/skypeforlinux/skypeforlinux
 RESTRICT="mirror bindist strip" #299368
 
-RDEPEND="dev-libs/atk
-	dev-libs/expat
-	dev-libs/glib:2
-	dev-libs/nspr
-	dev-libs/nss
-	gnome-base/gconf:2
+RDEPEND="dev-libs/atk[${MULTILIB_USEDEP}]
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	gnome-base/gconf:2[${MULTILIB_USEDEP}]
 	gnome-base/libgnome-keyring
```

Comment:
I think it won't work 'correctly' on x32 because without multilib there still will be no /usr/lib64/libgnome-keyring.so available.

I deleted gnome-keyring and libgnome-keyring for test and skype starts normally but it asks for  login credentials every time because it uses gnome-keyring for storing that informations.